package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Sample of an OpenUI5 app",
   "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "postinstall": "bower install",
+    "ui5": "grunt"
```

Comment:
What about `start` so that you can just `npm start`? (see https://docs.npmjs.com/cli/start)